trunk/lib/utils/widgets-rendering-logic.php

```diff
@@ -42,3 +51,13 @@ function v3_render_bottom_line_widgets($v3_widgets_enables) {
 		add_action('woocommerce_single_product_summary', 'wc_yotpo_show_buttomline',7);	
 	}
 }
+
+// CATEGORY PAGE RENDER
+function v3_category_page_renders($v3_widgets_enables): void {
+	if ($v3_widgets_enables['reviews_carousel_category']) {
+		add_action('woocommerce_after_shop_loop', 'wc_yotpo_show_reviews_carousel_widget', 10);
```

Comment:
this name `wc_yotpo_show_reviews_carousel_widget` should be here the same as for 
```
if($v3_widgets_enables['reviews_carousel_product']) {
	add_action('woocommerce_after_single_product', 'wc_yotpo_show_reviews_carousel_widget', 10);
}
```
?

Reply:
it is meant to be placed in category page after products ngFor loop